README.spanish.md

```diff
@@ -17,12 +17,12 @@
  [![nodepractices](/assets/images/twitter-s.png)](https://twitter.com/nodepractices/) **¡Síguenos en Twitter!** [**@nodepractices**](https://twitter.com/nodepractices/)
  <br/>
 
-# ¡Bienvenido! 3 cosas que necesitas saber primero:
-**1. Cuando lees aquí, lees docenas de los mejores artículos de Node.JS -** este es un resumen y conservación del contenido mejor clasificado de las mejores prácticas de NodeJS
+# ¡Bienvenido! 3 cosas que deberias saber primero:
+**1. Aquí encontraras docenas de los mejores artículos sobre Node.JS -** este es un resumen y conservación del contenido mejor clasificado de las mejores prácticas de NodeJS
```

Comment:
Missing accent on "encontrarás"! Otherwise LGTM :smile:

Reply:
Got it, just fixed it, thanks